mlflow/store/tracking/file_store.py

```diff
@@ -484,12 +485,26 @@ def _hard_delete_run(self, run_id):
         _, run_dir = self._find_run_root(run_id)
         shutil.rmtree(run_dir)
 
-    def _get_deleted_runs(self):
+    def _get_deleted_runs(self, older_than=0):
+        """
+        Get all deleted run ids.
+        Args:
+            older_than: get runs that is older than this variable in number of milliseconds.
+                        defaults to 0 ms to get all deleted runs.
+        """
+        current_time = int(time.time() * 1000)
         experiment_ids = self._get_active_experiments() + self._get_deleted_experiments()
         deleted_runs = self.search_runs(
             experiment_ids=experiment_ids, filter_string="", run_view_type=ViewType.DELETED_ONLY
         )
-        return [deleted_run.info.run_uuid for deleted_run in deleted_runs]
+        deleted_run_ids = []
+        for deleted_run in deleted_runs:
+            _, run_dir = self._find_run_root(deleted_run.info.run_uuid)
+            meta = read_yaml(run_dir, FileStore.META_DATA_FILE_NAME)
+            if current_time - meta["delete_time"] > older_than:
```

Comment:
Nit: Can we call this `deleted_time` instead?

Reply:
:ok_hand: